examples/intel-optimized-mysql-server_vpc_creation_testing/main.tf

```diff
@@ -4,29 +4,37 @@
 
 # Provision Intel Optimized AWS MySQL server
 module "vpc" {
-  source = "github.com/intel/terraform-intel-aws-mysql"
-  name = "my-vpc"
-  cidr = "10.0.0.0/16"
+  source = "terraform-aws-modules/vpc/aws"
+  name   = "my-vpc"
+  cidr   = "10.0.0.0/16"
 
 
 
-  azs             = ["us-east-1a", "us-east-1b", "us-east-1c"]
+  azs             = ["us-east-2a", "us-east-2b", "us-east-2c"]
   private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
 
 
   tags = {
-    Terraform = "true"
-    Environment = "dev"
+    Owner    = "Intel.Cloud.Optimization.Modules@inte.com"
+    Duration = "4"
   }
 }
 
 module "optimized-mysql-server" {
-  source         = "github.com/intel/terraform-intel-aws-mysql"
+  source         = "intel/aws-mysql/intel"
   rds_identifier = "mysql-dev"
```

Comment:
I would add a version constraint in here. It could be relatively forgiving where it's `~> 1.0.1` so that you are just allowing minor version changes

Reply:
Thanks! Done.